test/01LessonOneTest.js

```diff
@@ -2,39 +2,60 @@
 
 var assert = require('assert');
 
+//Homework: read through lesson one. Complete the lesson 1 test. Read the pigLatin.
 
 function returnTrue() {
     // should return true
-
+    return true;
 }
 
 function returnFalse() {
     // should return false
-
+    return false;
 }
 
 function greaterThan(big, small) {
     // should return true if the first argument is greater than the second argument
     // and return false if the first argument is less than the second argument
-
+    if(big > small) {
```

Comment:
@dorianpenaloza Have you had success running the tests? Anything I can help with?